### DISCUSSION TITLE
How to contribute to the project?

### DISCUSSION THREAD

**Discussion by Chensokheng** [react-icons/react-icons#580] [Q&A]
Is there any document on how to make the contribution to the project?

For example:
- how to run project locally.
- style guideline
- .....

**Comment by AlexMNet** [comment]
@Chensokheng  There is a Contributing section in the readme. [https://github.com/react-icons/react-icons#Contributing](https://github.com/react-icons/react-icons#Contributing)

I tried to build the project locally early and no luck.
